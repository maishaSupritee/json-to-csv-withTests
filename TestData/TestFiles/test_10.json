[
    {
        "animal":"\t"
        
    },
    {
        "animal":"\n"
      
    },
    {
        "breed":"\r"
    
    },
    {
        "coat pattern":"\b"
  
    }
  ]