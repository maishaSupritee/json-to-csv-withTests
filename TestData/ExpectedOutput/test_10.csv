"fruit_animal","fruit_breed","fruit_coat pattern"
"	","",""
"
","",""
"","",""
"","",""
